pkg/placement/controller.go

```diff
@@ -75,17 +75,20 @@ var excludedResourceNames = map[string]bool{
 type Controller struct {
```

Comment:
general comment, not on your code but on this file (probably Paolo's code):
I didn't understand why the excluded versions and resources are stored in map and not in a set (or array, or list).
the question is resource/version should be excluded is if set contains it or not. there is no meaning for the true/false value here.